docs/reference/rollup/rollup-getting-started.asciidoc

```diff
@@ -1,7 +1,10 @@
 [role="xpack"]
 [testenv="basic"]
 [[rollup-getting-started]]
-== Getting Started
+=== Getting started with {rollups}
+++++
```

Comment:
Ultimately, I think this is an example of a place we don't want to use the "Getting started" terminology.